plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -237,8 +237,12 @@ class Conan(
             return
         }
 
+        parseConanRemoteList(remoteList.stdout)
```

Comment:
The function may be a bit misleading, because the function is not just parsing something but has side effects.

I believe it would be nice to have a function which only returns `remoteUrl` / `remoteName` but includes line 234. 
```
data class RemoteList(url: String, name: String)

fun getRemoteList(): RemoteList {
        // List configured remotes in "remotes.txt" format.
        val remoteList = run("remote", "list", "--raw")
        ...
}
```
What do you think? (Maybe type of `url` could already be `Uri` so that the caller can simply do `runCatching` to have all error cases covered)

Reply:
I've removed the side effect, though without introducing a separate `data class` for simplicity.

I'm deliberately not including the call to `conan remote list ...` into that function as my plan is not use another call here for Conan 2 support.